packages/fxa-settings/src/components/App/RememberPassword/en.ftl

```diff
@@ -0,0 +1 @@
+pw-remember-pw = Remember password? Sign in
```

Comment:
I know this is migrating a gettext string, but could you add a comment for context? It's not 100% clear that this is a link users can follow to sign in.

Also in the mockup there's other strings, so confirm whether or not those also need to be added as ftl strings.

Reply:
Thank you! Added a comment.